### ISSUE TITLE
Tool tip information which gets displayed upon hovering on button but not displayed with keyboard focus.

### ISSUE THREAD

**Issue by msftedad** [microsoft/fast-blazor#3823]
# 🐛 Bug Report

Tool tip information which gets displayed upon hovering on button but not displayed with keyboard focus.
Keyboard dependent users will find it difficult to navigate as they will be unaware about the tooltip present.

## 💻 Repro or Code Sample

**Steps to Repro:**

1. Open URL: [Try Fluent UI Blazor - Write, compile, execute and share Fluent UI Blazor component compositions in the browser](https://try.fluentui-blazor.net/snippet/mEwJYpaWGeRJqbkr) in latest edge browser.
2. Press tab key to reach 'Test button'.
3. Verify whether when keyboard focus moves to the 'button' tooltip is visible or not.

## 🤔 Expected Behavior
When keyboard focus moves to the 'button' tooltip as Testing. is should be visible.

## 😯 Current Behavior
When keyboard focus moves to the 'button' tooltip is not visible. When hovered with mouse, Tooltip as Testing is Visible.

https://github.com/user-attachments/assets/25329e6c-196e-4b08-b07f-b30f9e1efcd9

https://github.com/user-attachments/assets/f49f3edd-1a07-4812-a0e2-7410eda17a11

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/b964b8f7-4ead-4616-ba6e-dccb34b793cf" />

## 🔦 Context

Keyboard dependent users will find it difficult to navigate as they will be unaware about the tooltip present.
When keyboard focus moves to the 'button' tooltip as Testing. is should be visible.

## 🌍 Your Environment

* OS & Device: Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26100.4061)
* Browser Edge Version: 136.0.3240.76 (Official build) (64-bit)
* .NET Version [Powered by .NET 9.0.4](https://dotnet.microsoft.com/en-us/learn/aspnet/what-is-aspnet-core) and Fluent UI Blazor library Version 1.0.0-alpha-1+7841f252

**Comment by DHoussama** [comment]
@vnbaaij  is there an ETA when v5 will be available?

**Comment by vnbaaij** [comment]
No, we never give out ETAs. It is done when it's done.

**Comment by dvoituron** [comment]
But you can already used the Preview version and give us your feedback. 

https://fluentui-blazor-v5.azurewebsites.net/installation

**Comment by DHoussama** [comment]
@dvoituron  Thanks , looks great. May I ask why do we need interactivity for this to work ? Can't it work with static Server?

**Comment by dvoituron** [comment]
> [@dvoituron](https://github.com/dvoituron) Thanks , looks great. May I ask why do we need interactivity for this to work ? Can't it work with static Server?

In v5 the Tooltips work with Static rendering.

![Image](https://github.com/user-attachments/assets/48b3bbb7-6b6d-40d3-9292-532f4d7857ce)
